docs/develop/dapps/payment-processing/accept-payments-in-a-telegram-bot-js.md

```diff
@@ -0,0 +1,440 @@
+# Bot for the sale of dumplings
+
+
+In this article, we'll create a simple Telegram bot for accepting payments in TON coins.
+
+## 🦄 What it looks like
+
+The bot will look like this:
+
+![bot preview](https://raw.githubusercontent.com/ton-community/ton-docs/main/static/img/tutorials/js-bot-preview.png?raw=true)
```

Comment:
One more about the image: better to make Image wide, so anyone from PC will see good picture instead of "mobile-screen-img".

Good wide img: https://www.tonspace.co/develop/dapps/payment-processing/accept-payments-in-a-telegram-bot-2

Bad (will be fixed soon): https://www.tonspace.co/develop/dapps/payment-processing/accept-payments-in-a-telegram-bot#bot-in-action